### DISCUSSION TITLE
Print Info

### DISCUSSION THREAD

**Discussion by Dakweed666** [OrcaSlicer/OrcaSlicer#4245] [Dev]
Is there anyway to add this automatically when you slice a file to the start of the g-code? It provides in for on the screen when you select a file to print, this applies to as far as I know most Ender printers, mine in particular is a V3 SE Cura adds it but I can't get Orca to do it, this is the lines it needs to be able to add. It's the very first 4 lines in the g-code.

;FLAVOR:Marlin
;TIME:2125
;Filament used: 2.21685m
;Layer height: 0.2

**Comment by beyondsoft978** [comment]
The same situation applies to Windows. How can I solve it?
